### PR TITLE
additional props added

### PR DESCRIPTION
**Pull request by harleenkohli**

### CODE REVIEW

File: src/Sarif/Schemata/sarif-schema.json [hunk 1]
```diff
@@ -2107,6 +2107,26 @@
...
+          "description": "A product suite to which the tool belongs.",
+          "type": "string"
+        },
+
+        "shortDescription": {
+          "description": "A brief, multiformat description of the tool.",
+          "$ref": "#/definitions/multiformatMessageString"
+        },
+
+        "longDescription": {
```
Comment (lgolding) [inline src/Sarif/Schemata/sarif-schema.json]: >longDescription [](start = 9, length = 15)

Sorry, the issue was wrong. s/b `fullDescription` to match the corresponding property name in `reportingDescriptor`.

File: src/Sarif/Schemata/sarif-schema.json [hunk 2]
```diff
@@ -2107,6 +2107,26 @@
...
+          "type": "string"
+        },
+
+        "product": {
+          "description": "A product suite to which the tool belongs.",
+          "type": "string"
+        },
+
+        "shortDescription": {
+          "description": "A brief, multiformat description of the tool.",
```
Comment (lgolding) [inline src/Sarif/Schemata/sarif-schema.json]: >multiformat [](start = 35, length = 11)

I don't think the word "multiformat" is needed in the description. Just "A brief description of the tool".

**Comment by lgolding** [comment]
Approval subject to my two comments.

File: src/Sarif/Schemata/sarif-schema.json (hunk 1 above)
Comment (michaelcfanning) [reply src/Sarif/Schemata/sarif-schema.json]: good catch

---
In reply to: [261382346](https://github.com/Microsoft/sarif-sdk/pull/1315#discussion_r261382346) [](ancestors = 261382346)

**Pull request merged**